.devcontainer/post_create.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# This script assumes we are running in a Miniconda container where:
+# - /opt/conda is the Miniconda installation directory
+# - https://github.com/conda/conda is mounted at /workspaces/conda
+# - https://github.com/conda/conda-libmamba-solver is mounted at
+#   /workspaces/conda-libmamba-solver
+
+set -euo pipefail
+
+CONDA_SRC=${CONDA_SRC:-/workspaces/conda}
+CONDA_LIBMAMBA_SOLVER_SRC=${CONDA_LIBMAMBA_SOLVER_SRC:-/workspaces/conda-libmamba-solver}
+
+# Clear history to avoid unneeded conflicts
+echo '' > /opt/conda/conda-meta/history
```

Comment:
Hm. How do we distinguish unneeded conflicts from the needed ones?

Reply:
This is to avoid issues like "you installed python=3.8 in this base env, and now the solution asks me for python=3.10, but you didn't say that explicitly, so ha! you are stuck forever with py38 til you write python=3.8 ex-pli-cit-ly!". It's like starting with no soft pins.

Reply:
We will still get conflicts if the things we are asking for are not compatible with each other.